distribution/lib/Standard/Base/0.0.0-dev/src/Data.enso

```diff
@@ -20,14 +20,19 @@ import project.System.File_Format.File_Format
 
 from project.Data.Boolean import Boolean, True, False
 
-## Read a file using the specified file format
+## ALIAS Load, Open
+   Read's a file into Enso.
+   Uses the specified file format to parse the file into an Enso type. If not
+   specified will use the file's extension to determine the file format.
 
    Arguments:
    - path: The path to the file to read.
    - format: A `File_Format` object used to read file into memory.
      If `Auto_Detect` is specified; the provided file determines the specific
      type and configures it appropriately. If there is no matching type then
      a `File_Error.Unsupported_Type` error is returned.
+     You can use `File_Format.all` to get a list of currently available
```

Comment:
```suggestion
     You can use `File_Format.all` to get a list of currently loaded
```

Available sounds a bit like "supported" or installed. But this will load formats from libraries which are currently imported, right?